src/coord/src/catalog/builtin.rs

```diff
@@ -997,7 +995,6 @@ lazy_static! {
             .with_column("key_group", ScalarType::Int64.nullable(false))
             .with_key(vec![0, 1, 4]), // TODO: explain why this is a key.
         id: GlobalId::System(4003),
-        index_id: GlobalId::System(4004),
```

Comment:
should we change the system table ids to also include the even ids now? we're only using the old ids after 4000 now which might be surprising to future readers of this code?

Reply:
It's definitely surprising, but it is too late to change the IDs! Their values may be hardcoded in the definition of various other views in the system.